pages/common/az-config.md

```diff
@@ -0,0 +1,21 @@
+# az config
+
+> Manage Azure CLI configuration.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Get all configurations:
+
+`az config get`
+
+- Get configurations for a section:
+
+`az config get {{section_name}}`
+
+- Set a configuration:
+
+`az config set {{key_value}}`
+
+- Unset a configuration:
+
+`az config unset {{key}}`
```

Comment:
```suggestion
`az config unset {{key_value}}`
```

Any reason why these 2 are different (the one above)

Reply:
Yes, the main reason is when you `set` a config, you have to provide the value, for example:
```bash
az config set defaults.group=myResourceGroup
```
The key would be `defaults.group` and the value `myResourceGroup`.

When unsetting a vaule (with `unset`), you only have to provide the key and not the value:
```bash
az config unset defaults.group
```

I'm not very sure what other words may clear the difference... Any suggestions?

Reply:
Makes somewhat sense to me :+1:

Reply:
Thanks! Should I resolve the conversation?

Reply:
I've done it :)